### PR TITLE
changed json files to flask endpoints

### PR DESCRIPTION
**Pull request by Southwick-Adam**

### CODE REVIEW

**Comment by Southwick-Adam** [comment]
I solved the merge issues. please take a look then merge

File: app.py
Comment (denartha10) [inline app.py:1]: appreciate the updates to the endpoints will add cache later

File: static/index.js
Comment (denartha10) [inline static/index.js:1]: I can add a linting rule to keep only single quotes

**Review by denartha10 — APPROVED** [review]
lovely

**Comment by denartha10** [comment]
no

**Pull request closed without merging**